cli/utils.go

```diff
@@ -40,16 +41,13 @@ func GetAlertmanagerURL(p string) url.URL {
 // parseMatchers parses a list of matchers (cli arguments).
 func parseMatchers(inputMatchers []string) ([]labels.Matcher, error) {
 	matchers := make([]labels.Matcher, 0, len(inputMatchers))
-
```

Comment:
drive-by/nit: There's a function in this file `GetAlertmanagerURL` that is also not used, can you remove it as part of this change? If not, I'll create a separate PR for it.

Reply:
[Remove unused function GetAlertmanagerURL #3535](https://github.com/prometheus/alertmanager/pull/3535)